protocol/avdl/keybase1/upk.avdl

```diff
@@ -111,4 +111,26 @@ protocol UPK {
       case V1: UserPlusAllKeys;
       case V2: UserPlusKeysV2AllIncarnations;
   }
+
+  enum UpkLiteMinorVersion {
+      V0_0
+  }
+
+  // only latest incarnation
+  record UpkLitev1 {
+      UID uid;
+      string username;
+      Seqno eldestSeqno;
+      StatusCode status;
+      // Sorted by generation ascending
+      map<KID, PublicKeyV2NaCl> deviceKeys;
+  }
+
+  record UpkLitev1AllIncarnations {
```

Comment:
Isn't `V` capital on the heavy version? Follow whatever template we have there.

Reply:
oops. fixed.